tests/src/python/test_qgsserver_wms_getfeatureinfo.py

```diff
@@ -259,7 +259,6 @@ def testGetFeatureInfoRelationReference(self):
                                  'test_project_values.qgz')
 
     # TODO make filter work with gpkg and move test inside testGetFeatureInfoFilter function
```

Comment:
Is the TODO here still open or was that what you solved?

Reply:
Oh yes, you're right, I have to remove this comment... Thanks :)